public/docs/swagger.yaml

```diff
@@ -414,644 +412,468 @@ definitions:
     Crop:
         type: object
         properties:
-            IDX_Crop:
-                type: number
-                description: to be updated
-                example: 2322356
             IDX_Property:
                 type: number
-                description: "to be updated"
+                description: to be updated
                 example: 1971349
             Crop_Variety:
                 type: string
-                description: "to be updated"
+                description: to be updated
                 example: "Not Stated"
             Crop_Name:
                 type: string
-                description: "to be updated"
+                description: The name of a crop
                 example: "Ackee"
             Crop_Total_Area:
                 type: number
-                description: "to be updated"
+                description: The total area of land the crop is grown on
```

Comment:
Once any changes are made within the schema we have, they will be reflected here. These reflect it's current state.